server/index.ts

```diff
@@ -114,6 +116,23 @@ io.on('connection', socket => {
 		}
 	});
 
+	socket.on(SocketEvent.VOTE_TO_CLEAR, async () => {
+		try {
+			const result = await RoomController.voteToClearBoard(socket.id);
+			if (result === null) {
+				return;
+			}
+			const { room, cleared } = result;
+			io.to(room.roomCode).emit(SocketEvent.VOTE_TO_CLEAR_ADDED, {
+				connectionId: socket.id,
```

Comment:
Can we also send information for who all has voted to clear? That way the UI can be updated with a count and an indicator next to their name